doc: peripheral UART device name fix (#17677)
nrfconnect/sdk-nrf: samples/bluetooth/peripheral_uart/README.rst

         If you use Thingy:53, you must attach the debug board and connect an external USB to UART converter.
      #. |connect_terminal|
      #. Reset the kit.
      #. Observe that **LED 0** is blinking and the device is advertising under the default name **Nordic_UART_Service**.

>          This name can be configured in the :kconfig:option:`CONFIG_BT_DEVICE_NAME` Kconfig option.

Reviewer: ```suggestion
         You can configure this name using the :kconfig:option:`CONFIG_BT_DEVICE_NAME` Kconfig option.
```
